src/modules/nodejs.rs

```diff
@@ -33,6 +33,30 @@ pub fn module<'a>(context: &'a Context) -> Option<Module<'a>> {
     }
 
     let nodejs_version = Lazy::new(|| {
+        #[cfg(all(windows, not(test)))]
+        {
+            use crate::modules::utils::version_win;
+            let resolved_node_path = match which::which("node") {
+                Ok(p) => p,
+                Err(e) => {
+                    log::error!("Failed to resolve path for node: {:?}", e);
```

Comment:
Maybe this shouldn't be logged as an error since this will occur if the user does not have node installed.